gulp/config.js

```diff
@@ -33,21 +33,16 @@ module.exports = () => {
     layouts: {
       src: `${basePaths.src}layouts/`,
     },
-    images: {
-      src: `${basePaths.content}images/`,
-      dest: `${basePaths.dest}content/images/`,
+    assets: {
+      src: [
+        `${basePaths.src}materials/icons/`,
+        `${basePaths.content}images/`
```

Comment:
One last thing, can we add:
- assets/images
- assests/fonts

here and the folders with gitkeep so that we guide the dev also in the structure to understand where those should go?